src/cryptography/hazmat/backends/interfaces.py

```diff
@@ -221,6 +221,12 @@ def elliptic_curve_exchange_algorithm_supported(self, algorithm, curve):
         Returns whether the exchange algorithm is supported by this backend.
         """
 
+    @abc.abstractmethod
+    def derive_elliptic_curve_public_point(self, private_value, curve):
```

Comment:
Sorry I didn't get to this before; what's the rational for having the backend interface return `(x, y)` instead of a `ECPrivateKey` instance?